packages/core/src/browser/quick-open/prefix-quick-open-service.ts

```diff
@@ -145,13 +146,23 @@ export class PrefixQuickOpenService {
     @inject(QuickTitleBar)
     protected readonly quickTitleBar: QuickTitleBar;
 
+    @inject(BrowserMenuBarContribution)
+    protected readonly browserMenuBarContribution: BrowserMenuBarContribution;
+
     /**
      * Opens a quick open widget with the model that handles the known prefixes.
      * @param prefix string that may contain a prefix of some of the known quick open handlers.
      * A default quick open handler will be called if the provided string doesn't match any.
      * An empty quick open will be opened if there's no default handler registered.
      */
     open(prefix: string): void {
+        if (this.browserMenuBarContribution.menuBar) {
+            const activeMenu = this.browserMenuBarContribution.menuBar.activeMenu;
```

Comment:
it would be better to store menuBar variable before if instead, instead of querying it twice